geonode/documents/templates/layouts/doc_panels.html

```diff
@@ -261,6 +264,24 @@
                     <div class="row">
                         <div class="col">
                             <div id="basicInfo" class="container-fluid">
+                                <div>
+                                    <img id="image">
+                                </div>
+                                <script>
+
+                                    const image = document.getElementById('image');
+                                    const path = location.pathname.split('/');
+                                    const id = path[path.length - 2];
+                                    const prefetch_url = location.origin + "{{resource.curatedthumbnail.thumbnail_url}}";
```

Comment:
why don't you use {{ static ... }} here instead of calculating the URL by yourself?

Reply:
im not sure if that is static url. here is an example:
/uploaded/CACHE/images/curated_thumbs/blob_BKSWXbj/0513cd62f2b53208f9a75dc1c48e6766.png

Reply:
I't automatically manahed by static. I told you to give a look to thumbnail_upload.html. You would have seen that static works fine there

Reply:
done